src/SesWebhookCall.php

```diff
@@ -20,4 +20,10 @@ public static function storeWebhook(WebhookConfig $config, Request $request): We
             'payload' => $message->toArray(),
         ]);
     }
+
+    public function identicalMessageExists() {
+        return SesWebhookCall::where('id', '<>', $this->id)
```

Comment:
To solve the problem you mentioned, maybe we can check if the currently handled webhookcall id is the lowest id for the `MessageId` being handled?

If it is, we should process the webhook, if it isn't, we shouldn't process the webhook.

Reply:
Agreed.I'll put that in

Reply:
SesWebhookCall's identicalMessageExists() method is now isFirstOfThisSesMessage()